### DISCUSSION TITLE
Can users create their own accounts or at least register for approval?

### DISCUSSION THREAD

**Discussion by TechMitten** [wavelog/wavelog#999] [Q&A] [answered]
I was hoping to use this in my app, but I don't want to create accounts for people. Are there self-care options:?

**Comment by HB9HIL** [comment]
Wavelog is an open-source software. Check our Wiki for Installation Procedure on a Webserver.

**Comment by HB9HIL** [comment]
PS: Wavelog does NOT support public registration for other users.

**Comment by TechMitten** [comment]
I installed it on my server, but I'll read through it again.

-------- Original Message --------

**Comment by HB9HIL** [comment]
So you are asking for some kind of public registration? So everyone can create an account by himself?

We do not support this yet

**Comment by TechMitten** [comment]
Yes, otherwise I'm confused on how this works. Is this for friends of Hams? It just appears to be a very high quality, comprehensive program made to scale...but it comes down to the server operator and who they might know with their license.

-------- Original Message --------

**Reply by HB9HIL** [reply, marked as answer]
The most common case is that hams which wan't to use Wavelog, install their own instance. Sure you can create accounts for your friends and other hams. But currently
you have to do this manually. 

The case of a "public instance" where strangers can register freely is not really common and we had no plans yet to implement a registration form.

**Reply by TechMitten** [reply]
> The most common case is that hams which wan't to use Wavelog, install their own instance. Sure you can create accounts for your friends and other hams. But currently you have to do this manually.
> 
> The case of a "public instance" where strangers can register freely is not really common and we had no plans yet to implement a registration form.

I totally get it and I don't blame you for being cautious about who uses the app in a licensed hobby. Although, for a logging program, probably not a big deal to let anyone join. It would be really easy to have AI check for official FCC licenses and reject or approve based upon that. If you think about it, that's what others do, for software like Echo link etc. They check the uploaded license and give them the go based on it.

**Reply by int2001** [reply]
Probably a misunderstanding here.
We provide Software. For Download / Self-Install.

We do **not** provide Wavelog as a hosted Software or as a Service.
Everyone who installed the software is responsible for his own users. Therefore no "AI-Checking" or "Registration-Portal" is planned.

So if you installed Wavelog and want some of your friends using it as well then add them via the Admin-Menue as User. Job done.

**Comment by TechMitten** [comment]
Got it. Sorry for my mistake with the suggestion.

-------- Original Message --------
